docs/usepatterns.rst

```diff
@@ -21,3 +21,35 @@ there are use patterns to maximizing your productivity.
     # The full version, including alpha/beta/rc tags.
     release = REVER_VERSION
 
+==================
+Initializing Rever
+==================
+There are a couple steps you should take to get the most out of rever.
+1. Install rever. Rever is on conda-forge so install via ``conda install rever -c conda-forge`` or from source.
```

Comment:
rst needs blank line above numbered list

Reply:
Also, you can install rever from pypi so it should also be mentioned that you can do ``pip install re-ver``

Reply:
And this line is too long